manifests/k/KurtZimmermann/RegCoolX64/1.324/KurtZimmermann.RegCoolX64.installer.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: KurtZimmermann.RegCoolX64
+PackageVersion: 1.324
+InstallerLocale: en-us
+InstallerType: exe
+InstallerSwitches:
+  Silent: /Si
+  SilentWithProgress: /Si
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://kurtzimmermann.com/files/winget/RegCoolX64.exe
+  InstallerSha256: 38D9DD1AD7E1E2D8733FDE47D280DF7DE17B84228DC30404C2ADA9D1D0D85253

```

It seems like the .exe has been changed since the validations ran.

```suggestion
  InstallerSha256: 4998F7E3B2B208E046BF261E311042515B7694B6E843DE656F1B9588993A726E
```

---

@Info445, do you have a link that points to a specific version instead?